src/JuMP.jl

```diff
@@ -738,5 +729,26 @@ include("print.jl")
 # Nonlinear-specific code
 include("nlp.jl")
 
+
+typealias _getTypes Union{NonlinearExpression,NonlinearParameter, AffExpr}
+function getValue{T<:_getTypes}(arr::Array{T})
```

Comment:
Couldn't we just define this as ``map(getValue,arr)``? I don't think this is performance critical code and the performance difference from using map will be disappearing soon.